.github/workflows/go-test.yaml

```diff
@@ -0,0 +1,53 @@
+name: NATS Server Testing
+on: [push, pull_request]
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        go: [1.13, 1.14]
+
+    env:
+      GOPATH: /home/runner/work/nats-server
+      GO111MODULE: "off"
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v1
+        with:
+          path: src/github.com/nats-io/nats-server
+
+      - name: Setup Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{matrix.go}}
+
+      - name: Install deps
+        shell: bash --noprofile --norc -x -eo pipefail {0}
+        run: |
+          go get github.com/nats-io/nats.go/
```

Comment:
Let's try to transition to modules for these tests or use vendor, but not a third option.

Reply:
While I agree with what you're proposing, I think there's value in seeing how it behaves next to travis in a like for like situation.

That's what you wanted to see :) Once we're happy I think your suggestions are good next steps but it would be interesting over a few weeks doing a like for like compare

Reply:
That is fair. Can I get my badge though for quick access?

Reply:
added a badge but it'll probably be broken for a while - first pr doesnt run the tests

Reply:
Why does it not run them?

Reply:
They have access to secrets and various things, so you want to only run them once project maintainers approves.

Reply:
https://github.com/nats-io/nats-server/actions/runs/111413323

Reply:
Not following, can take offline. I just want tests to run on any branch and PR etc like Travis.

Reply:
they will run, its only the PR that initially ads them for the first time (ie. this one) that doesnt have them run.